photonVision/cameras/Microsoft_LifeCam_HD-3000/pipelines/Cargo Red.json

```diff
@@ -0,0 +1,86 @@
+[ "ColoredShapePipelineSettings", {
+  "pipelineIndex" : 1,
+  "pipelineType" : "ColoredShape",
+  "inputImageFlipMode" : "NONE",
+  "inputImageRotationMode" : "DEG_0",
+  "pipelineNickname" : "Cargo Red",
+  "cameraExposure" : 49.1,
+  "cameraBrightness" : 50,
+  "cameraGain" : -1,
+  "cameraVideoModeIndex" : 0,
+  "streamingFrameDivisor" : "NONE",
+  "ledMode" : true,
+  "inputShouldShow" : true,
+  "outputShouldShow" : true,
+  "hsvHue" : {
+    "first" : 0,
+    "second" : 20
+  },
+  "hsvSaturation" : {
+    "first" : 80,
+    "second" : 200
+  },
+  "hsvValue" : {
+    "first" : 114,
+    "second" : 255
+  },
+  "outputShouldDraw" : true,
+  "outputShowMultipleTargets" : true,
+  "contourArea" : {
+    "first" : 1.5,
+    "second" : 100.0
+  },
+  "contourRatio" : {
+    "first" : 0.0,
+    "second" : 20.0
+  },
+  "contourFullness" : {
+    "first" : 0.0,
+    "second" : 100.0
+  },
+  "contourSpecklePercentage" : 51,
+  "contourSortMode" : "Largest",
+  "contourTargetOffsetPointEdge" : "Center",
+  "contourTargetOrientation" : "Landscape",
+  "offsetRobotOffsetMode" : "None",
+  "offsetSinglePoint" : {
+    "x" : 0.0,
+    "y" : 0.0
+  },
+  "offsetDualPointA" : {
+    "x" : 0.0,
+    "y" : 0.0
+  },
+  "offsetDualPointAArea" : 0.0,
+  "offsetDualPointB" : {
+    "x" : 0.0,
+    "y" : 0.0
+  },
+  "offsetDualPointBArea" : 0.0,
+  "contourGroupingMode" : "Single",
+  "contourIntersection" : "Up",
+  "solvePNPEnabled" : false,
+  "targetModel" : "k2020HighGoalOuter",
+  "cornerDetectionStrategy" : "APPROX_POLY_DP_AND_EXTREME_CORNERS",
+  "cornerDetectionUseConvexHulls" : true,
+  "cornerDetectionExactSideCount" : false,
+  "cornerDetectionSideCount" : 4,
+  "cornerDetectionAccuracyPercentage" : 10.0,
+  "contourShape" : "Circle",
+  "contourPerimeter" : {
+    "first" : 0.0,
+    "second" : 4000.0
+  },
+  "accuracyPercentage" : 10.0,
+  "circleDetectThreshold" : 15,
+  "contourRadius" : {
+    "first" : 10,
+    "second" : 10
+  },
+  "minDist" : 20,
```

Comment:
> Circle Match Distance - How close the centroid of a contour must be to the center of the circle in order for them to be matched. This value is usually pretty small (<25) as you usually only want to identify circles that are nearly centered in the contour.

I believe what this means is how close must the center of the circle be to the center of the bounding box in pixels.

<img width="251" alt="image" src="https://user-images.githubusercontent.com/3326002/152277041-fb31593c-b2ca-45b8-9baf-5b138f4ff564.png">

<20 pixels is probably fine here. I just wanted to clarify the meaning of this value for future reference.